boards/px4/fmu-v5x/init/rc.board_sensors

```diff
@@ -48,50 +49,51 @@ then
 	fi
 fi
 
-if ver hwtypecmp V5X90 V5X91 V5X92 V5Xa0 V5Xa1 V5Xa2
+if ver hwtypecmp V5X00 V5X01 V5X02 V5X10 V5X40 V5X41 V5X42
 then
-	#SKYNODE base fmu board orientation
+	#FMUv5Xbase board orientation
 
-	if ver hwtypecmp V5X90 V5X91 V5Xa0 V5Xa1
+	if ver hwtypecmp V5X00 V5X01 V5X40 V5X41
 	then
 		# Internal SPI BMI088
-		bmi088 -A -R 2 -s start
-		bmi088 -G -R 2 -s start
+		bmi088 -A -R 4 -s start
+		bmi088 -G -R 4 -s start
 	else
 		# Internal SPI bus ICM20649
-		icm20649 -s -R 4 start
+		icm20649 -s -R 6 start
 	fi
 
 	# Internal SPI bus ICM42688p
-	icm42688p -R 4 -s start
+	icm42688p -R 6 -s start
 
 	# Internal SPI bus ICM-20602 (hard-mounted)
-	icm20602 -R 8 -s start
+	icm20602 -R 10 -s start
 
 	# Internal magnetometer on I2c
-	bmm150 -I -R 6 start
+	bmm150 -I start
 
 else
-	#FMUv5Xbase board orientation
+	#SKYNODE base fmu board orientation
 
-	if ver hwtypecmp V5X00 V5X01
+	# V5X010001 ommited because not representable in this hardware format
```

Comment:
I'm not sure about this.

Reply:
I have a bad feeling about this, but I'm unsure. We are using V5X010001 (VER 0x10, REV 0x01), that's actually the current gen. Just because VER 0x10 is read from EEPROM (according to the ADC it would be ver 07x), it doesn't change the startup script, right? I believe this would break the build for Skynode

Reply:
@nicovanduijn so this change is only for the `release/v1.13` branch, and will go into v1.13.1. On `main` this should all be fine. Does that resolve it?

My problem was that I don't know how `V5X010001` would map back to the old `V5Xabc` format.

Reply:
it's still conted as base16, we just skipped 0b,0c,0d,0e,0f and went to 0x10 after 0x0a to leave space for the remaining resistor space.
e.g. 0x00 - 0x0f : resistors
       0x10 - 0xff: eeprom
       
at least that's my understanding of how it went. 
But I can't tell you either how 0x010001 maps to the v5xabc format. I guess that just doesn't work, so we'll have to live with the fact that it won't boot on skynode

Reply:
Ok, got it, thanks! This is only v1.13, so I assume it doesn't really matter for you/Auterion anyway.

Reply:
> it's still conted as base16, we just skipped 0b,0c,0d,0e,0f and went to 0x10 after 0x0a to leave space for the remaining resistor space. e.g. 0x00 - 0x0f : resistors 0x10 - 0xff: eeprom
> 
> at least that's my understanding of how it went. But I can't tell you either how 0x010001 maps to the v5xabc format. I guess that just doesn't work, so we'll have to live with the fact that it won't boot on skynode

There are 11 code point that can be read from resistors. 
0 - 0xb (0-11) are the values. 0x7 means read the EEPROM
In the EEPROM with current print format the using 3 digits each  (see [HW_INFO_REV_DIGITS and HW_INFO_VER_DIGITS](https://github.com/PX4/PX4-Autopilot/blob/e3dff5c07450e89b6c859b1403f6b1d0c26cd36c/platforms/common/include/px4_platform_common/board_common.h#L268-L269) [and](https://github.com/PX4/PX4-Autopilot/blob/dff946c39a01b883fbd7967c8b9255ca02be826b/platforms/nuttx/src/px4/common/include/px4_platform/board_determine_hw_info.h#L38) 
the value are 0x10-0x3E7 (16-999)  

Values are displayed and compared in HEX  as  `lllvvvrrr` where lll is the label (V5, V6X DH etc) vvv is 3 hex digits for version and rrr is 3 hex digits for revision.

On X series the VER is the Base Board and the Rev is the FMUM.

In the code the #defines for the HW configs may use the abbreviated where thay fit.
`#define V5X01 HW_VER_REV(0x0,0x1)` that is V5X0000001`
This was done to prevent coupling if we compiled with HW_INFO_xxx_DIGITS 2 or 3 or 5 as the value are still valid.